src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/tests/CCInstructions.unit.spec.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderWithStoreAndRouter } from '~/platform/testing/unit/react-testing-library-helpers';
+import { Toggler } from '~/platform/utilities/feature-toggles';
+import CCInstructions from '../CCInstructions';
+
+const appointmentData = {
+  start: '2024-07-19T08:00:00-07:00',
+  location: {
+    vistaId: '983',
+    clinicId: '848',
+    stationId: '983',
+    clinicName: 'CHY PC VAR2',
+  },
+};
+
+describe('CCInstructions component', () => {
+  const initialState = {
+    featureToggles: {
+      [Toggler.TOGGLE_NAMES.vaOnlineSchedulingDescriptiveBackLink]: true,
+    },
+  };
+
+  it('should render comment in CCInstructions', async () => {
+    const appointment = {
+      ...appointmentData,
+      comment: 'Follow-up/Routine: I have a headache',
+    };
+
+    const wrapper = renderWithStoreAndRouter(
+      <CCInstructions appointment={appointment} />,
+      {
+        initialState,
+      },
+    );
+
```

Comment:
You can also add a block to check for presence of `<h2>`
```suggestion
    expect(
      await wrapper.queryByRole('heading', {
        level: 2,
        name: 'Special instructions',
      }),
    ).to.exist;

```